desktop/app/notifications.js

```diff
@@ -0,0 +1,38 @@
+import engine from '../../react-native/react/engine'
+import Window from './window'
+
+// Electron doesn't currently support showing notifications from the main
+// process, so we'll use a hidden window to show them for us.
+// https://github.com/atom/electron/issues/3359
+const window = new Window('notifier', { show: false })
+window.show()
+
+function notify (title, opts) {
+  window.window.webContents.send('notify', title, opts)
+}
+
```

Comment:
Can we use the existing code [here](https://github.com/keybase/client/blob/master/react-native/react/native/notifications.desktop.js) here.